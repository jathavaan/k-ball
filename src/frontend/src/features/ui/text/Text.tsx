import { StyledText } from "./text.style.ts";
import { TextProps } from "./text.types.ts";

export const Text = (props: TextProps) => {
  return (
    <StyledText {...props}> 
    {props.text}
  </StyledText>
  );
};
