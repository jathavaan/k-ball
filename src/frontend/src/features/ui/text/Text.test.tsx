import { render } from "@testing-library/react";
import { Text } from "./Text";

it("should match Text snapshot", () => {
  const { asFragment } = render(
    <Text
      onClick={function (): void {
        throw new Error("Function not implemented.");
      }}
      text={""}
    />,
  );
  expect(asFragment()).toMatchSnapshot();
});
