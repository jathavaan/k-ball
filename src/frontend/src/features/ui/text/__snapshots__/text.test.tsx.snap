// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should match ArrowDownwardIcon snapshot 1`] = `
<DocumentFragment>
  <p
    class="MuiTypography-root MuiTypography-body1 css-1r3a445-MuiTypography-root"
    text=""
  />
</DocumentFragment>
`;

exports[`should match Text snapshot 1`] = `
<DocumentFragment>
  <p
    class="MuiTypography-root MuiTypography-body1 css-1r3a445-MuiTypography-root"
    text=""
  />
</DocumentFragment>
`;
